make/development-environments.mk

```
@@ -0,0 +1,122 @@
+##@ Deployment
+
+GATEWAYAPI_PROVIDER = ISTIO
+GATEWAYAPI_PROVIDER_LOWERCASE = $(shell echo $(GATEWAYAPI_PROVIDER) | tr A-Z a-z)
```

Can we make the expected value lowercase? We're not using `ISTIO` (uppercase) anywhere, are we?

```suggestion
GATEWAYAPI_PROVIDER = istio
```

---

:+1:  done